turn/__init__.py

```diff
@@ -216,7 +240,6 @@ class TurnMessageTemplates(TurnBusinessManagementRequest):
     endpoint_name = "message_templates"
 
     def get_message_templates(self):
-        self.method = "GET"
```

Comment:
I think this is still needed though it would seem like a better design to pass the method into the `do_request` function instead of setting it as a property on the class.

Reply:
Good catch. This line should not have been deleted. Seeing that this is touched, I'll go ahead an make the update to pass the method to `do_request`. It's a quick win

Reply:
https://github.com/dimagi/turn-python/pull/3/commits/d949ec3f6e1b34c134550396f5f8495a3f28c84b